api/lib/request/uploadFileFormData.ts

```diff
@@ -0,0 +1,58 @@
+const request = require('request-promise')
```

Comment:
Again could we use axios with form-data? See https://codeburst.io/how-to-handle-multipart-form-data-in-nodejs-file-uploading-in-nodejs-26c0cb88adcf (not read it thoroughly though, just know it's possible)

Reply:
Discussed with Alan, we time boxed this, and it went over 2hrs, as upload will most likely be thrown away I've used request-promise in this separate file, and we can easily remove the file later, leaving only Axios within the project :)